### DISCUSSION TITLE
Trial_ends_after_video

### DISCUSSION THREAD

**Discussion by rbruno1** [jspsych/jsPsych#2110] [Q&A] [answered]
I know there is a plug in for trial_ends_after_audio, but my experiment uses videos. Is there something similar for video experiments?

**Comment by jodeleeuw** [comment, marked as answer]
Yes, the video plugins support `trial_ends_after_video`.

See the [docs here](https://www.jspsych.org/plugins/jspsych-video-keyboard-response/#parameters).
